plugins/prometheus_plugin/include/eosio/prometheus_plugin/prometheus_plugin.hpp

```diff
@@ -14,7 +15,7 @@ namespace eosio {
       prometheus_plugin();
       ~prometheus_plugin();
 
-      APPBASE_PLUGIN_REQUIRES((http_plugin) (chain_plugin) (net_plugin))
+      APPBASE_PLUGIN_REQUIRES((http_plugin) (chain_plugin) (net_plugin) (producer_plugin))
```

Comment:
`chain_plugin`, `net_plugin`, `producer_plugin` are all required plugins so doubt it matters, but in general I don't think you need to require any plugin except `http_plugin`.